util/bigint/src/hash.rs

```diff
@@ -605,25 +610,26 @@ mod tests {
 	}
 
 	#[test]
+	#[ignore]
 	fn shift_bloomed() {
-		use sha3::Hashable;
+		//use sha3::Hashable;
```

Comment:
I should have mentioned that in PR description. It's commented out and will be moved out of `hash.rs`. These tests use keccak function and I didn't want to bring sha3 as a dependency here.